src/app/parcels/batch/BatchParcelsReducer.ts

```diff
@@ -35,128 +34,128 @@ const getOverridenFieldsAndValues = (
         );
 };
 
-function createBooleanGroupFromStrings(strings: string[] | null): BooleanGroup {
-    const result: BooleanGroup = {};
-    if (strings) {
-        strings.forEach((str) => {
-            result[str] = true;
-        });
-    }
-    return result;
-}
+// function createBooleanGroupFromStrings(strings: string[] | null): BooleanGroup {
+//     const result: BooleanGroup = {};
+//     if (strings) {
+//         strings.forEach((str) => {
+//             result[str] = true;
+//         });
+//     }
+//     return result;
+// }
```

Comment:
If these functions aren't currently required we don't need to comment them out, we can just not call them. Also, similarly to the above, I think it might be worth moving these to a `reducerHelpers.ts` file to keep things compartmentalised